frontend/android/src/main/java/io/github/droidkaigi/confsched2019/ui/MainActivity.kt

```diff
@@ -122,6 +122,11 @@ class MainActivity : DaggerAppCompatActivity() {
             binding.logo.isVisible = config.isShowLogoImage
             if (!config.hasTitle) supportActionBar?.title = ""
 
+            if (destination.id == R.id.session_detail) {
```

Comment:
can we define this property in config class? like `val hideToolbar: Boolean` as well as `config.hasTitle`.

Reply:
I agreed. I fixed it.